api/cases/case-list.js

```diff
@@ -61,7 +61,9 @@ module.exports = (req, res, next) => {
                 return casesData;
             })
         .then(casesData => {
-        let results = rawCasesReducer(casesData, sscsCaseListTemplate.columns).sort(function (result1, result2) {
+        const results = rawCasesReducer(casesData, sscsCaseListTemplate.columns)
+            .filter(row => row.case_reference != undefined && row.case_reference != null)
```

Comment:
Needs to be `!==` instead of !=`

Reply:
Sure, will make the changes to use !== instead of !=